eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -172,7 +171,7 @@ instance EraPParams era => FromCBOR (ConwayGovState era) where
   fromCBOR = fromEraCBOR @era
 
 data EnactState era = EnactState
-  { ensCommittee :: !(StrictMaybe (Set (KeyHash 'Voting (EraCrypto era)), Rational))
+  { ensCommittee :: !(StrictMaybe (Map (Credential 'CommitteeColdKey (EraCrypto era)) EpochNo, Rational))
```

Comment:
Now looking at the code, makes me realize it would be a good idea to create a type for Committee (especially since tuples are lazy and we want our state to be fully strict:
```suggestion
  { ensCommittee :: !(StrictMaybe (Committe era)
```

where

```haskell
data Committee era = Committee 
  { committeeMembers :: !(Map (Credential 'CommitteeColdKey (EraCrypto era)) EpochNo)
  , committeeQuorum :: !Rational
  }
```

Then this type can also be used in the `NewCommittee` type:

```haskell
  | NewCommittee
      -- | Old committee
      !(Set (Credential 'CommitteeColdKey (EraCrypto era)))
      -- | New Committee with expiration epoch and quorum
      !(Committee era)
```

Reply:
Implemented